osl/sandbox.go

```diff
@@ -166,6 +166,11 @@ type Interface interface {
 	// and moving it out of the sandbox.
 	Remove() error
 
+	// SetSecondaryIP adds or deletes the passed IP as a secondary IP on
+	// the interface. for ex: set the vip of services in the same network
+	// as secondary IP.
+	SetSecondaryIP(ip *net.IPNet, add bool) error
```

Comment:
I think that it would be nice to keep the current flow and do not add a new setter method in this interface which only has getter methods.

I am referring to intead modify the `Sandbox` interface at the top of this file adding, next to the existing `AddInterface()` method a new `UpdateInterface(SrcName string, option IfaceOption, add bool) error` method which takes one only `IfaceOption` and a flag specifying it is an add or a remove.

The osl sandbox inplementaiton is free to only accept a `IPAliases` options setter on the updateinterface path and refuse any other as not implemented yet.